.github/workflows/main-build.yml

```diff
@@ -0,0 +1,123 @@
+name: Main Build
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-20.04
+    name: Integration Test - ${{ matrix.name }}
+    strategy:
+      matrix:
+        include:
+# TODO(anuraaga): Enable after opentelemetry-lambda changes applied.
```

Comment:
Depends on https://github.com/open-telemetry/opentelemetry-lambda/pull/44 and https://github.com/open-telemetry/opentelemetry-lambda/pull/45

Reply:
update Python https://github.com/open-telemetry/opentelemetry-lambda/pull/47